### PR TITLE
add 1.0.0 version info and download link

### PR DESCRIPTION
**Pull request by wanganjuan**
image

### CODE REVIEW

File: content/en/docs/download/download.md [hunk 1]
```diff
@@ -6,14 +6,33 @@ weight: 2
...
...z.sha512)] | [[Source](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0-src.tar.gz)] [[Sign](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0-src.tar.gz.asc)] [[SHA512](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0-src.tar.gz.sha512)] | [[Source](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-computer-incubating-1.0.0-src.tar.gz)] [[Sign](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-computer-incubating-1.0.0-src.tar.gz.asc)] [[SHA512](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-computer-incubating-1.0.0-src.tar.gz.sha512)] | [[Source](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-commons-incubating-1.0.0-src.tar.gz)] [[Sign](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-commons-incubating-1.0.0-src.tar.gz.asc)] [[SHA512](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-commons-incubating-1.0.0-src.tar.gz.sha512)] | [Release-Notes](/docs/changelog/hugegraph-1.0.0-release-notes/) |
```
Comment (imbajin) [inline content/en/docs/download/download.md:28]: remember update the `release` date

File: content/cn/docs/download/download.md [hunk 2]
```diff
@@ -6,16 +6,31 @@ weight: 2
 
 ### Latest version
 
-> 当前 HugeGraph: 1.0.0 正在投票中，完成后更新当前页面。
+The latest HugeGraph: **1.0.0**, released on _2023-02-22_([how to build from source](/docs/quickstart/hugegraph-server/)).
 
+| components          | description                                      | download                                                                                                           |
+| ------------------- | ------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------ |
+| HugeGraph-Server    | HugeGraph 的主程序                               | [1.0.0](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-incubating-1.0.0.tar.gz)           |
+| HugeGraph-Toolchain | 数据导入/导出/备份, Web 可视化图形界面等工具合集 | [1.0.0](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0.tar.gz) |
```
Comment (imbajin) [inline content/cn/docs/download/download.md:14]: remember add cdn site as backup

File: content/cn/docs/download/download.md [hunk 3]
```diff
@@ -6,16 +6,31 @@ weight: 2
...
...z.sha512)] | [[Source](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0-src.tar.gz)] [[Sign](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0-src.tar.gz.asc)] [[SHA512](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-toolchain-incubating-1.0.0-src.tar.gz.sha512)] | [[Source](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-computer-incubating-1.0.0-src.tar.gz)] [[Sign](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-computer-incubating-1.0.0-src.tar.gz.asc)] [[SHA512](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-computer-incubating-1.0.0-src.tar.gz.sha512)] | [[Source](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-commons-incubating-1.0.0-src.tar.gz)] [[Sign](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-commons-incubating-1.0.0-src.tar.gz.asc)] [[SHA512](https://downloads.apache.org/incubator/hugegraph/1.0.0/apache-hugegraph-commons-incubating-1.0.0-src.tar.gz.sha512)] | [Release-Notes](/docs/changelog/hugegraph-1.0.0-release-notes/) |
```
Comment (imbajin) [inline content/cn/docs/download/download.md:26]: update date

File: content/en/docs/download/download.md (hunk 1 above)
Comment (wanganjuan) [reply content/en/docs/download/download.md]: image
fixed

File: content/cn/docs/download/download.md (hunk 2 above)
Comment (wanganjuan) [reply content/cn/docs/download/download.md]: image
fixed

File: content/cn/docs/download/download.md (hunk 3 above)
Comment (wanganjuan) [reply content/cn/docs/download/download.md]: fixed

**Review by imbajin — APPROVED** [review]

**Pull request merged**